Views/Players/PlayerInfo.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using dotnetNewProject.Models
+@model NewPlayer
+
+<ul>
+    @foreach (var player in ViewData["players"] as IList<NewPlayer>)
+    {
+        <li>
+            @player.PlayerFirstName @player.PlayerLastName @player.Age
+            <hr>
+
+        </li>
+    }
+    <div>
+
+        <li>
+            @TempData["firstName"] @TempData["lastName"]
+        </li>
+    </div>
+    <hr>
+
+
+
+    <html>

```

Comment:
Each view will be rendered inside of a `Layout` - which already has a `<html>` tag, and we can nest document tags.  You might see these from examples when you're searching for stuff that's rendering out a whole page, but when including in your project, you usually just want to grab the `body` contents.